_posts/2021-01-08-status-update.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: "2021-01-08: IAC Team Status Update"
+author: pmcmaw
+categories:
+  - team
+  - status
+tags:
+  - community
+  - releases
+  - dsc
+  - windows
+  - cloud-ci
+  - puppet-module-gem
+  - rubocop
+  - infrastructure
+  - networking
+  - puppetize
+  - triage
+---
+
+## Happy New Year!
+
+On behalf whole IAC team we would like to take the opportunity to wish all readers a Happy New Year! We hope you all had a great Christmas. The team took the opportunity enjoy the holidays, recharge our batteries and are now ready to tackle 2021 head on.
+
+![2021]({% link /assets/2020-01-08-status-update/2021.jpg %})
+
+## Well Hello TP!
```

Comment:
"Happy New TP" to fit in with the rest of the headers? :rofl:

Reply:
Niceeeee..... Good shout :-)